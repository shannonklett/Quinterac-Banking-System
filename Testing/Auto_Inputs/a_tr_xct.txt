login
agent
transfer
123456
223456
99999999
logout