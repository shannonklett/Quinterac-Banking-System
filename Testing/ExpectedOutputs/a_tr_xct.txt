03 123456 223456 99999999                
00 000000 000000 00000000                
